README.md

```diff
@@ -104,39 +104,91 @@ dropdown boxes.
 
 #### Selecting Machine Types
 
-The easiest way to get a list of all known Machine Types is with:
+The easiest way to get a list of all known Machine Type name is by utilizing either of the two APIs:
+
+#### Old API:
+
+```
+cli.get_machine_type_names(clean_strings_out)
+```
+
+#### New API:
```

Comment:
What does old vs new api actually mean?  Write that down somewhere in the doc

Reply:
Done